config.yaml

```diff
@@ -13,8 +13,16 @@ remote_user: arteria
 remote_destination: /tmp/runfolders/
 remote_sisyphus_location: /opt/arteria/arteria-siswrap-env/deps/sisyphus/sisyphus-latest
 
-ngi_pipeline_url: http://nestor1.uppmax.uu.se:6666/test_flowcell_analysis
+ngi_pipeline_url: https://irma1.uppmax.uu.se:4444/ngi_pipeline_upps
```

Comment:
I agree with you that we should refactor Arteria some day... Even though this is not sensitive information there's a lot of SNP&SEQ specific stuff here now..